src/ripple/app/main/GRPCServer.cpp

```diff
@@ -429,7 +429,7 @@ GRPCServerImpl::GRPCServerImpl(Application& app)
     // if present, get endpoint from config
     if (app_.config().exists("port_grpc"))
     {
-        Section section = app_.config().section("port_grpc");
+        const auto& section = app_.config().section("port_grpc");
 
```

Comment:
Sneak in a minor improvement.

Reply:
The change you made here definitely is an improvement.  You're now taking a reference instead of making a copy.  And a `Section` is a heavy weight thing to copy.  Good spotting.

However, not everyone will agree that replacing an explicit type name with `auto` is an improvement.  In my opinion it hides important information that could be visible explicitly in the source code.  My personal opinion is that Herb Sutter was dead wrong when he said "almost always `auto`" years ago.  So my preferred declaration here would look like...
```
Section const& section = ...
```
Folks who use an IDE with something like intellisense sometimes lose track of those of us who who develop code using just a text editor.  Explicit type names (as opposed to `auto`) also are a big help when viewing code through browsers, like on Github or in code reviews.

I'm not asking for a change here.  Just providing a different perspective for you to think about when you make other changes in the future.